elastic/kibana
# [Maps] record hexagon clustering layer in telemetry

Working with @teresaalvarezsoler with telemetry cluster and realized telemetry had not been updated for hexagon clustering.

To test
* create map with hexagon clustering
* Open Kibana advanced settings
* Scroll to `Usage Data` section. 
* Open browser dev tools network tab
* Click `cluster data` 
* Copy response from `_stats` request
* Paste response in code editor
* Verify layerTypes.es_agg_hexagons accurately reflects the number of hexagon clustering layers existing in your saved objects.
